eng/platform-matrix.yml

```diff
@@ -32,15 +32,18 @@ jobs:
         osIdentifier: Linux
         containerName: ubuntu_1404_arm_cross_build_image
         helixQueues:
-          # Ubuntu.1404.Arm32.Open hardware is not capable of dealing with all PRs
-          ${{ if and(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI')) }}:
-            asString: 'Ubuntu.1404.Arm32.Open'
-            asArray:
-            - Ubuntu.1404.Arm32.Open
-          ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-            # We don't have any Linux/arm32 internal Helix queues
-            asString: ''
-            asArray: []
+        - ${{ if and(eq(variables['System.TeamProject'], 'public'), eq(variables['Build.Reason'], 'PullRequest')) }}:
+          - (Ubuntu.1804.Arm32.On.Docker.Open)Ubuntu.1604.Arm32.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm32v7-0a0ebdd-20190312215532
+        - ${{ if and(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI')) }}:
+          - Ubuntu.1404.Arm32.Open
```

Comment:
Can you add a comment explaining the separation of these two things. On first glance I would assume the split would not happen, but the logic for why is sound to me. Either way, I would prefer for the reasoning to be discoverable.

Reply:
Sure